### ISSUE TITLE
Encourage use of __matmul__ operator @ over np.dot / ndarray.dot?

### ISSUE THREAD

**Issue by effigies** [nipraxis/textbook#6]
Not critical to do today, but I've noticed that there are a number of places where the `@` operator would result in code that looks more like the equations in the text. Any objection to adding it to the outer-and-dot product discussion and mass-replacing in later texts? I know its semantics are not identical to `np.dot` in all cases (scalars and 3+ dimensions), but for vector/matrix math, I think it will always do what we want. Not sure if there are arguments against it...

**Comment by matthew-brett** [comment]
I agree that @ is best - it's just that it was new around the time of the course.  So please feel free to de-dot.

**Comment by matthew-brett** [comment]
Maybe put in the `_todo.md` file (I committed it a little while ago).

**Comment by effigies** [comment]
Sounds good. Figured it was mostly age.

**Comment by effigies** [comment]
> Maybe put in the `_todo.md` file (I committed it a little while ago).

Not seeing `_todo.md`, though it's in gh-pages. Did it not get pushed?

**Comment by matthew-brett** [comment]
Oops - sorry - pushed now.

**Comment by effigies** [comment]
In the todo.
